README.md

```diff
@@ -163,280 +173,41 @@ const order: MarketOrderSpec = {
 })().catch(console.error);
```

Comment:
Looks like this example isn't updated (maybe some merge error), it should be `glm.oneOf({ order })` instead of `oneOf(order)`